test/InterchainTokenService.js

```diff
@@ -1179,14 +1179,39 @@ describe('Interchain Token Service', () => {
     describe('Send Token', () => {
         const amount = 1234;
         const destAddress = '0x5678';
-        let token, tokenId;
+        let token, tokenId, tokenManager;
+        let payload, wrappedPayload, payloadHash, transferToAddress;
+
+        const buildPaylods = async () => {
+            payload = defaultAbiCoder.encode(
```

Comment:
not a fan of functions that changed variables in an outside scope as a side effect. try to keep code more functional in nature

Reply:
ok, let's address it in a follow up so we don't accumulate it as tech debt